### PR TITLE
Fix alignment of mine counts, simplfy code a bit.

### PR DESCRIPTION
**Pull request by mdakin**

### CODE REVIEW

File: examples/mine_digger/lib/main.dart [hunk 1]
```diff
@@ -19,18 +19,21 @@ import 'package:sky/src/fn3.dart';
...
+  const Color(0xFF13A023), // green
+  const Color(0xFFDA1414), // red
+  const Color(0xFF1E2347), // black
+  const Color(0xFF7F0037), // dark red
+  const Color(0xFF000000),
+  const Color(0xFF000000),
+  const Color(0xFF000000),
 ];
 
+final List<TextStyle> textStyles = textColors.map((c) =>
```
Comment (abarth) [inline examples/mine_digger/lib/main.dart]: `s/c/color/`

Please use complete words in variable names.

**Comment by abarth** [comment]
LGTM

File: examples/mine_digger/lib/main.dart [hunk 2]
```diff
@@ -19,18 +19,21 @@ import 'package:sky/src/fn3.dart';
...
+  new TextStyle(color: c, fontWeight: bold, textAlign: TextAlign.center)).toList();
```
Comment (Hixie) [inline examples/mine_digger/lib/main.dart]: only use fat arrow if you can do it without a line break.

File: examples/mine_digger/lib/main.dart [hunk 3]
```diff
@@ -259,25 +259,21 @@ class MineDiggerState extends State<MineDigger> {
...
     return count;
   }
 
-  int bombs(int x, int y) {
-    return cells[y][x] ? 1 : 0;
-  }
+  int bombs(int x, int y) => inBoard(x, y) && cells[y][x] ? 1 : 0;
+
+  bool inBoard(int x, int y) =>
+    (x > 0 && x < (cols - 1) && y > 0 && y < (rows -1));
```
Comment (Hixie) [inline examples/mine_digger/lib/main.dart]: ditto

**Comment by Hixie** [comment]
this is great, thanks!

**Comment by mdakin** [comment]
Thanks, I will send antoher PR to address the comments.

**Pull request merged**